SimG4CMS/Calo/src/HFShower.cc

```diff
@@ -221,7 +222,7 @@ std::vector<HFShower::Hit> HFShower::getHits(const G4Step *aStep, bool forLibrar
   double tSlice = (aStep->GetPostStepPoint()->GetGlobalTime());
 
 #ifdef EDM_ML_DEBUG
-  double time = (ignoreTimeShift_) ? 0 : fibre_->tShift(localPos, depth, chkFibre_);
+  double time = (equalizeTimeShift_) ? 0 : fibre_->tShift(localPos, depth, chkFibre_);

```

Comment:
@abdoulline, since I don't understand the details, I'm just looking for symmetries. Could you please confirm that here and at L329 below `time` must be `0` when `equalizeTimeShift_` is  `true`, and not `fibre_->tShift(localPos, depth, -1)` as elsewhere?

Reply:
@perrotta 
yes, in the aforementioned HFShower.cc there are 3 methods  HFShower::getHits  which assign such time term -
 
(1) full Cherenkov light simulation:  the time should be always non-0  (to be in sync with HFShowerLibrary.cc )
(2) usage of HF ShowerLibrary:  the time here shoud be 0 for equalizeTimeShift=true (as it's non-0 in HFShowerLibrary.cc)
(3) not used in regular wfs.